src/uct/api/uct.h

```diff
@@ -1056,6 +1056,13 @@ ucs_status_t uct_rkey_release(const uct_rkey_bundle_t *rkey_ob);
  * @ingroup UCT_RESOURCE
  * @brief Flush outstanding communication operations on an interface.
  *
+ * Flushes all outstanding communications issued on the interface prior to
+ * this call. The operations are completed at the origin or at the target
+ * as well. The exact completion semantic depends on the provided parameters.
```

Comment:
I would suggest to be more explicit and mention that: the semantics depends on @a flags parameter.